src/modules/sub-items/module-usage.md

```diff
@@ -0,0 +1,96 @@
+# Sub Items List
+
+Sub Items List module is meant to be used as a part of editorial interface of eZ Platform. It provides an interface for listing sub items list of any location.
```

Comment:
`Sub Items List module is meant to be used as a part of the editorial interface of eZ Platform.`
Also, I'd suggest either `an interface for listing the sub items of any location` or `an interface for displaying the sub items list of any location`